elfutils/patches/libdw-dwarf-frame-register.patch

```diff
@@ -0,0 +1,12 @@
+--- libdw/dwarf_frame_register.c
```

Comment:
do you have any source this patch is coming from?

if you do, could you please add that as URL to the beginning of the `.patch` file? (it gets ignored by `patch` command)

Reply:
Just to jump in here, I went to pick up this issue again and saw this PR, thanks for polishing it off @rwunderer!

This patch I initially built from a compilation error but it looks like a similiar one is on this mailing list https://www.mail-archive.com/elfutils-devel@sourceware.org/msg02896.html

Reply:
Thanks for the source links, @charlie-haley - saved me a lot of research time! :smiley: